product/tutorials/tutorial_import_products_gcs.md

```diff
@@ -68,53 +68,69 @@ The operation is completed when the field **```operation.done()```** is set to t
 
 Check the result, one of the following fields should be present:
  - **```error```**, if the operation failed.
- - **```response```**, if the operation was successful.
+ - **```result```**, if the operation was successful.
 
-You have imported valid product objects to the catalog. The operation should be successful, and the operation object should contain a **```response```** field. 
+You have imported valid product objects to the catalog.
 
-TODO[PUT OPERATION OBJECT HERE]
+Check the ```gcs_operation.metadata.success_count``` field to get the total number of the successfully imported products.
 
-Check the **```response```** field in the operation object returned to and printed in the Terminal. 
+The number of failures during the product import is returned in ```gcs_operation.metadata.failure_count``` field.
 
-The **```error_samples[]```** field should be empty.
+The operation is successful, and the operation object contains a **```result```** field.
+Check it printed out in a Terminal, it should be like following: 
 
-## Error handling
+```
+errors_config {
+  gcs_prefix: "gs://products_catalog/error"
+}
+```
+
+## Errors appeared during the importing
 
 Now, let's try to import a few invalid product objects and check the error message in the operation response. Note that in this case the operation itself is considered successful.
 
-The title field is required, so if you remove it, you get the invalid product object. There is an **```invalid_products_for_import.json```** in the Cloud Storage bucket containing such an invalid product.
+The title field is required, so if you remove it, you get the invalid product object. 
+The other example of invalid product is a product with some incorrect value of product.availability field.
+There is an **```products_for_import_some_invalid.json```** in the Cloud Storage bucket containing such an invalid products.
```

Comment:
1. replace "There is an" with "There is a"
2. add "file" before "in the Cloud Storage bucket"